app/views/aeon_pages/_info_modal.html.erb

```diff
@@ -0,0 +1,31 @@
+<div id="aeon-info-overlay" class="confirmation-overlay">
+  <div class="modal-body">
+    <h1 class="dialog-title" id='dialogTitle'><%= t('.header', library: Settings.libraries[i18n_library_title_key].label) %></h1>
+    <p><%= t('.reading_room_info') %></p>
+    <h2 class="dialog-how-to"><%= t('.how_to.header') %></h2>
+    <p><%= t('.how_to.body') %></p>
+    <p><%= t('.steps.header') %></p>
+    <ol class="dialog-steps">
+      <% if current_request.finding_aid? %>
+      <%= t('.steps.finding_aid_html') %>
+      <% elsif current_request.holdings.many? %>
+      <%= t('.steps.multi_vol_html') %>
+      <% else %>
+      <%= t('.steps.single_vol_html') %>
+      <% end %>
+    </ol>
+    <p class="dialog-more-info">
+      <%= t('.more_details_html', library: Settings.libraries[i18n_library_title_key].label, reading_room_url: Settings.libraries[i18n_library_title_key].reading_room_url) %>
+    </p>
+    <div class="dialog-buttons">
+      <% if current_request.finding_aid? %>
+        <%= link_to t('.continue_button'), current_request.finding_aid, class: 'btn btn-md btn-primary' %>
+      <% elsif current_request.all_holdings.many? %>
+        <%= button_tag(t('.continue_button'), class: 'btn btn-md btn-primary', type: 'button', data: { behavior: 'close-overlay', 'close-target' => '#aeon-info-overlay' }) %>
+      <% else %>
+        <%= submit_tag t('.continue_button'), class: 'btn btn-md btn-primary', form: 'new_request' %>
```

Comment:
What's does the `form: 'new_request'` option do?

Reply:
allows this button to submit the underlying (hidden) form directly, by targeting it with the [`form` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#form), while not needing to be located within the markup of the form.